src/httpServer.ts

```diff
@@ -170,26 +176,80 @@ function configureServer(httpServer: restify.Server) {
 
     // APIs - no html - just json response
 
-    httpServer.get('/api/v1.0/tasks', async (req, res, next) => {
-        await graphGet(req, res, next, `https://graph.microsoft.com/beta/me/outlook/tasks?${app.graph.filterNotCompletedAndNagMeCategory}&${app.graph.queryExpandNagExtensions}`);
+
+    httpServer.get('/api/v1.0/me', async (req, res, next) => {
+        await graphGet(req, res, next, "https://graph.microsoft.com/v1.0/me");
```

Comment:
feels like some of these endpoints should be configuration based, not inline in code.

Reply:
Agree #42